src/components/PreviewComment/index.js

```diff
@@ -110,18 +110,18 @@ export const styles = StyleSheet.create({
   lineBeforeProfile: {
     height: 9,
     borderLeftWidth: 1,
-    borderLeftColor: COLORS.gray1,
+    borderLeftColor: '#C4C4C4',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color '#C4C4C4' should be replaced with a constant for better maintainability.
```diff
- borderLeftColor: '#C4C4C4',
+ borderLeftColor: colors.lightGrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->